wiki/Tournaments/GBC/GBC_2024_Autumn/en.md

```diff
@@ -0,0 +1,280 @@
+---
+tags:
+  - GBC
+  - GB Cup
+  - GBC2024
+  - GBC 2024
+  - GBC2024Autumn
+  - GBC 2024 Autumn
+---
+
+# GB Cup 2024 Autumn
+
+![GBC 2024 Autumn banner](img/banner.png)
+
+The **GB Cup 2024 Autumn** (***GBC 2024 Autumn***) is a 3v3 captain draft–based osu!mania 4-key tournament hosted by Team GB. It is the seventh instalment of the GB Cup.
+
+## Tournament schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration phase | 2024-10-05/2024-10-20 (13:59 UTC) |
+| Qualifier showcase | 2024-10-20 (14:00 UTC) |
+| Qualifier stage | 2024-10-26/2024-10-27 |
+| Qualifier result showcase | 2024-11-03 (12:00 UTC) |
+| Drafting phase | 2024-11-10 |
+| Swiss I week 1 showcase | 2024-11-10 |
+| Swiss I week 1 | 2024-11-16/2024-11-17 |
+| Swiss I week 2 | 2024-11-23/2024-11-24 |
+| Swiss I week 3 | 2024-11-30/2024-12-01 |
+| Swiss II week 1 | 2024-12-07/2024-12-08 |
+| Swiss II week 2 | 2024-12-14/2024-12-15 |
+| Swiss II week 3 | 2024-12-21/2024-12-22 |
+| Elimination week 1 | 2024-12-28/2024-12-29 |
+| Elimination week 2 | 2025-01-04/2025-01-05 |
+
+## Organisation
+
+The GB Cup 2024 Autumn is run by Team GB and various community members.
+
+| Position | Member(s) |
+| :-- | :-- |
+| Host | ::{ flag=CN }:: [\[GB\]yobrevelc](https://osu.ppy.sh/users/14128407) |
+| Staff | ::{ flag=CN }:: [\[GB\]GanyuAngel](https://osu.ppy.sh/users/27233308), ::{ flag=CN }:: [\[GB\]Mafufu](https://osu.ppy.sh/users/10884561) |
+| Mappool manager | ::{ flag=CN }:: [\[GB\]yobrevelc](https://osu.ppy.sh/users/14128407), ::{ flag=CN }:: [\[GB\]Mafufu](https://osu.ppy.sh/users/10884561) |
+| Proposer | ::{ flag=KR }:: [\[GB\]SuddenDeath](https://osu.ppy.sh/users/6699923), ::{ flag=CN }:: [\[GB\]R-Trigger](https://osu.ppy.sh/users/25308131), ::{ flag=CN }:: [\[GB\]Thaumiel](https://osu.ppy.sh/users/15017771) |
```

Comment:
"proposer" doesn't make sense. is this meant to be "advisor"?

Reply:
it is mainly used to recommend existed maps for mappool, we never found such roles before so we use this word by translators, but it sounds quite different with advisor, would you have some better recommendation?

Reply:
so "mappool advisor"? if "proposer" is a direct translation of a role that assists in selecting maps for the mappool, then "advisor" would sound much better

Reply:
change to `mappool advisor`

Reply:
```suggestion
| Mappool advisor | ::{ flag=KR }:: [\[GB\]SuddenDeath](https://osu.ppy.sh/users/6699923), ::{ flag=CN }:: [\[GB\]R-Trigger](https://osu.ppy.sh/users/25308131), ::{ flag=CN }:: [\[GB\]Thaumiel](https://osu.ppy.sh/users/15017771) |
```